vault/core.go

```diff
@@ -2090,6 +2094,13 @@ func (c *Core) emitMetrics(stopCh chan struct{}) {
 			if c.expiration != nil {
 				c.expiration.emitMetrics()
 			}
+			//Refresh the sealed gauge
+			if c.Sealed() {
+				metrics.SetGauge([]string{"core", "unsealed"}, 1)
+			} else {
+				metrics.SetGauge([]string{"core", "unsealed"}, 0)
+			}
```

Comment:
I think the values here are switched, elsewhere unsealed==0 when c.Sealed() is true.

Reply:
Thanks guys, fixed above.